lib/vector-config/src/lib.rs

```diff
@@ -107,15 +107,15 @@
 // rules, or ... so on and so forth. and by concating the existing description on the shared type,
 // we can continue to include high-quality doc comments with contextual links, examples, etc and
 // avoid duplication.
-
+//
```

Comment:
```suggestion
```

Reply:
Hmm, no, this is an intentional line break to separate the TODOs.

Reply:
Gotcha, used to not seeing the comment syntax on an empty line.